docs/schedule/examples/fares-v1.es.md

```diff
@@ -0,0 +1,68 @@
+---
+search:
+  exclude: true
+---
+
+# Fares V1
+
+<hr/>
+
+Compuesto por [fare_attributes.txt](../../reference/#fare_attributestxt) y [fare_rules.txt](../../reference/#fare_rulestxt), Fares V1 ha sido históricamente el método oficial para describir la información de las tarifas en GTFS. Sin embargo, los dos archivos están limitados en cuanto a la amplitud de los factores que pueden describir eficazmente, y son ambiguos de implementar.[Fares V2](../../examples/fares-v2/) es un proyecto de extensión que se está desarrollando de forma activa y que pretende solucionar las limitaciones de Fares V1.
+
+## Definir las reglas tarifarias de una agencia
+
+Un trip en la red de metro de la Comisión de Tránsito de Toronto cuesta 3,20 dólares canadienses si los usuarios pagan con la tarjeta PRESTO. Los usuarios también pueden hacer transbordo a otras rutas de metro, tranvía o autobús operadas por la TTC en un plazo de dos horas.
```

Comment:
unsure if Toronto Transit Commission should have been translated, it wasn't in the German translation